docs/src/jsx/utils/codeSandboxUtils.js

```diff
@@ -7,5 +7,6 @@
 //  You may not use this file except in compliance with the License.
 
 export function inScreenshotTests() {
-  return navigator.userAgent.includes("HeadlessChrome");
+  // screenshot tests are not always in a headless Chrome, so need to check for "selectedKind" in the URL as well.
+  return navigator.userAgent.includes("HeadlessChrome") || window.location.search.includes("selectedKind");
```

Comment:
nit: URLSearchParams(window.location.search).has("selectedKind")

Reply:
Updated this to match other use agent tests